add mission CRF_SWP109_Schooners_End_Sortie to the repository

FluffSquirrel opened the pull request:
Mission Pull Request Template

This Pull Request will:

Add the mission Schooner's End Sortie made by muah to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Ingame Play Area Markers
- [x] Ingame Briefings to all players
- [x] Side-based briefings
- [x] Objective Entities
- [x] Mission .conf file

**Faction(s):**

OPFOR - Attacking
- [x] USSR

INDFOR - Defending
- [x] FIA

**Objectives/Win Conditions:**

ussr must seize mob orel. they can optionally seize fob sakal to neutralize mortars

**Terrain:**

everon

**Short mission description:**

USSR forces will attempt to dislodge well prepared FIA forces located at the Schooner's End military base.

**Slotting Ratio:**

<3:2> opfor:indfor

**Time limit:**

100m

**Mission Briefing Screenshot:**
![image](https://github.com/user-attachments/assets/b1b7cdc0-9c1d-4ce6-8ba9-2d03beddbb5c)

**Design concept/thoughts:**
it's lamentin limbo but infantry based.
obviously its a big ao with no vehicles to move, i feel like it will be fine but i also though LL would be fine and it really wasnt, so idk. i just like large scale infantry based missions, so i made one and if that means we arent going to play it soon cause we are getting average 50-60 vs 80 players to make the larger AO ops work, then thats fine by me. just enjoyed making it so it doesnt matter to me if it isnt played.

**Other Notes:**

<Assets? Admin notes? Anything else?>
assets:
2x mortars for fia & a camo uaz ambulance
2x pkm techys for USSr and an olive uaz ambulance

admin notes:
would be good to tp fia forces to positions they want to go to (inside Safestart border) before opstart. just to save time and braincells. i remember doing this for forest fuckery, dont think it should be too objectionable.

RandAlThor1 commented:
**QA Feedback**

- Coy doesn't need the 1 in 1coy same with indfor plt
<img width="193" alt="image" src="https://github.com/user-attachments/assets/fdc51aff-a7c7-47d5-9a47-376e6cf5c2dd">

- 2nd platoon is labeled company 
<img width="254" alt="image" src="https://github.com/user-attachments/assets/f848d4c4-b545-4af3-bd1c-016ae10824a5">

- ussr safe start border has a bit sticking out
<img width="103" alt="image" src="https://github.com/user-attachments/assets/313ef6da-e124-4843-a6a7-4af0e2b2f163">

- indfor briefing two typos
<img width="404" alt="image" src="https://github.com/user-attachments/assets/e057b3f3-99e7-4890-a48f-ccd4e7c698d3">

- maybe giving indfor a even just a single truck to help get people to and from their FOB will make it a more appealing objective for them to try and use.

FluffSquirrel commented:
qa feedback response:
mission updated:
- 1 removed from 1coy opfor, now coy
- 1 removed from 1plt indfor, now plt
- 2nd platoon opfor relabeled
- ussr safestart border moved to match moved spawns, i forgot to move the border when i moved the spawn. fixed bit sticking out.
- fixed typos indfor brief
- 1x ural4320 civilian added to fia, updated assets list, removed mention of GM tps in indfor brief
